### DISCUSSION TITLE
HiFiNi 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#54] [模板讨论]
HiFiNi 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by hecarli555** [comment]
请问大佬，变量v和val是干嘛用的？设置多少合适？

**Reply by wjf0214** [reply]
更新最新的，或者不用填就行。

**Reply by hecarli555** [reply]
好的明白

**Reply by hecarli555** [reply]
求大佬看到

**Comment by hecarli555** [comment]
貌似又跪了…签到日志提示这样
签到结果为：【】

网页登录，能看到没有签到

**Reply by hecarli555** [reply]
观察了一下官网…貌似是加入了人机识别机制

**Comment by zcb411711** [comment]
今天加入了人机识别机制

**Reply by wjf0214** [reply]
更了。

**Comment by hecarli555** [comment]
求大佬更新

**Reply by wjf0214** [reply]
再更一次吧。现在网站换的太频繁了。签的多了估计又得换验证。

**Reply by hecarli555** [reply]
今天它自动又签上了………奇奇怪怪的

**Reply by wjf0214** [reply]
签到上了。那就要不是没盾了，要不还是之前的盾。

**Comment by Wirtz-7** [comment]
网站换了新的验证，希望大佬有空更新一下，感谢
<img width="560" alt="image" src="https://github.com/wjf0214/qd-templates/assets/124484047/53830a8e-a8e9-4c95-bcea-94dcee4c25ca">

**Comment by hecarli555** [comment]
我也是提示存在风险。。。。

**Comment by hecarli555** [comment]
签到已坏，求大佬更新

**Reply by wjf0214** [reply]
网站又改回去了吧？我看OK的。

**Reply by hecarli555** [reply]
好的，感谢大佬

**Comment by jialin4567** [comment]
报错了 签到结果为：【操作存在风险，请稍后重试。】

**Reply by wjf0214** [reply]
增加了个参数验证，先观察下吧，如果key不变的话，应该签到写死就行。这样到时候再更新。
否则单纯依靠签到框架没法生成参数。可能要改为使用写脚本的方式去执行。

**Reply by hecarli555** [reply]
等大佬修复

**Comment by hecarli555** [comment]
请问模板修好了么？
